components/dashboard-modules/NumberDisplay.tsx

```diff
@@ -10,15 +10,13 @@ interface Props {
 
 const NumberDisplay: FC<Props> = ({ title, value, onClick, isLoading }) => {
   //isLoading = !isLoading;
-  const Element = onClick ? 'button' : 'div';
```

Comment:
What's the value in removing this `Element` system?

Reply:
Honestly I removed it as part of trying to figure out what the heck was going wrong—I can put it back in before merging, since I do like how it keeps things consistent

Reply:
Oh wait no I can't trivially put it back—if onClick is present, I'm wrapping the numberdisplay in a button

Reply:
Yeah this separation into a function works.